sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py

```diff
@@ -64,7 +64,8 @@
   submission_date
 """
 
-EXCLUDED_NAMESPACES = {"xfocsp_error_report"}  # restricted access
+# Restricted access
+EXCLUDED_NAMESPACES = {"regrets_reporter", "contextual_services"}
```

Comment:
I did local testing this time to make sure all the remaining issues were worked out. Because my personal account doesn't have access to these two namespaces, I got errors there. I think it's good to include these exemptions to keep this script runnable by data eng accounts.